README.md

```diff
@@ -1,5 +1,129 @@
 # hyp3-flood-monitoring
 
-## Python version
+## Architecture overview
 
-Requires Python 3.9.
+The [Pacific Disaster Center](https://www.pdc.org/about) (PDC)
+provides a Hazard API (see [PDC Hazard API](#pdc-hazard-api)).
+The purpose of this project is to provide RTC products for areas of interest affected by active flood hazards.
+This is accomplished by maintaining a [HyP3 subscription](https://hyp3-docs.asf.alaska.edu/using/subscriptions/)
+for each active flood hazard.
+
+An AWS Lambda function runs periodically and executes the following steps:
+
+<ol>
+<li>
+Query the PDC Hazard API for a list of active flood hazards.
+</li>
+<li>
+For each hazard:
+  <ol type="a">
+  <li>
+    If there is no existing HyP3 subscription for the hazard, create one, and set its end datetime for a few hours
+    into the future.
+  </li>
+  <li>
+    Otherwise, update the existing HyP3 subscription with any parameters that have changed (e.g. AOI),
+    and set its end datetime for a few hours into the future.
+  </li>
+  </ol>
+</li>
+</ol>
+
+Note that when a hazard expires (becomes inactive), the following steps occur automatically:
+
+1. The hazard disappears from the list of active hazards returned by the PDC Hazard API.
```

Comment:
Super minor, but why is this `1.` instead of HTML `< ol >`?

Reply:
Ah, I was only using HTML tags because markdown doesn't support lettered lists (A, B, C, etc.), but I didn't need that for this particular list (as I did for the one above). So was just easier to use normal markdown.